ext/spl/tests/bug66405.phpt

```diff
@@ -53,6 +53,8 @@ rmdir($td . '/testsubdir');
 rmdir($td);
 ?>
 --EXPECTF--
+Deprecated: Return type of Bug66405::current() should either be compatible with FilesystemIterator::current(): SplFileInfo|FilesystemIterator|string, or the #[ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in %s on line %d
```

Comment:
Why not add the type in this test?

Reply:
Ahh, I was just too tired when I did this.